examples/webgl_loader_gltf.html

```diff
@@ -97,6 +97,10 @@
 
 					scene.add( gltf.scene );
 
+				}, undefined, function ( e ) {
+
+					console.error( e );
+
```

Comment:
BTW, why you added `onError` function here? For the test? Or to be better as an example?

Reply:
People copy/paste this example to use with other models, and will ask on the forum on Stack Overflow when things don't work. GLTFLoader prints the error even if you don't give an error callback, but some loaders fail silently, so I think we should just include the error callback in examples anyway as best practice.

Reply:
I see I see. I agree. Just in case I wanted to know the purpose.